apps/web/pages/_document.tsx

```diff
@@ -3,10 +3,9 @@ import Document, { DocumentContext, Head, Html, Main, NextScript, DocumentProps
 type Props = Record<string, unknown> & DocumentProps;
 
 function toRunBeforeReactOnClient() {
-  const calEmbedMode =
-    location.search.includes("embed=") ||
-    /* Iframe Name */
-    window.name.includes("cal-embed");
+  const calEmbedMode = typeof new URL(document.URL).searchParams.get("embed") === "string";
```

Comment:
Make embed detection even more reliable by  using searchParams that automatically handles all cases like
- a.com/?embed
-  a.com/?embed=
- a.com/?b=1&embed=
- a.com/?b=1&embed&c